README.md

```diff
@@ -1 +1,21 @@
 # Requests
+
+# Draft of the Rules
+
+- Anyone can join @morganey-lang organization as a Member
+
+- If someone have an Open Sourced Morganey related project, they can join the organization as a Project Leader by transfering the ownership of the project to @morganey-lang. What that means:
+
+    - The Project Leader preserves all of the legal rights on the transfered project. From the technical point of view I will make sure that after the transfering the ownership the Project Leader has enough control over their repo.
```

Comment:
Honestly I'm not sure about legal stuff. You know, technically every contributor **have** legal rights on his own contribution. If the contributor wants to legally transfer their rights to some another entity — they **should** sign CLA of some form.

Usually we don't care about the legal stuff (because it sucks); personally I just define the copyright holder as some non-anthropomorphous entity like "Morganey contributors" that will automatically assimilate every contributor, so no problems will arise for a while.

I want to *discuss* whether we should include the legal stuff here at all (both in this point and the point "The Project Leader can transfer the project". I don't think we should make any legal assumptions or requirements here.

Although the one assumption I'd really like to add is a license requirement. We may ask the project leaders to relicense their project under one of the approved (by Morganey-lang organization) licenses, so we (as an organization) won't have any problems supporting the software in the future.

My personal list of approved licenses (and I'm ready to discuss any changes):

- MIT
- BSD
- WTFPL (and I'd like to relicense such a project under MIT after its landing into the organization)